ci/plugins/cloudtest/hooks/command

```diff
@@ -39,5 +39,8 @@ ci_collapsed_heading "kind: Purging state from previous builds..."
 bin/ci-builder run stable test/cloudtest/reset
 rm -f kubectl-*.log
 
+ci_collapsed_heading "kail: Start an instance"
+bin/ci-builder run stable --detach --name="kail" kail --context kind-cloudtest --log-level info --log-file "kail-output.log"
```

Comment:
make sure you do `docker kill kail || true` to clean up any existing kail instances.

Reply:
I had that before but was wondering why I still need it. Don't I get a new EC2 instance? Or is it a used one?

Reply:
Oh sorry, my fault, you do not need it, we already have a cleanup : 
```
ci_collapsed_heading ":docker: Purging all existing docker containers and volumes, regardless of origin"
docker ps --all --quiet | xargs --no-run-if-empty docker rm --force --volumes
```
earlier in the same file.

Buildkite keeps a cache of instances and your CI jobs may be given an instance that ran something previously that was not cleaned up completely.